src/Components/ProvisioningWizard/index.js

```diff
@@ -78,6 +78,7 @@ const ProvisioningWizard = ({ onClose, hasAccess, image, ...props }) => {
         onNext={onNext}
         className={'provisioning'}
         footer={<CustomFooter />}
+        closeButtonAriaLabel='Close launch wizard'
```

Comment:
Besides the cancel button we also have the global `X` button for closing the wizard.
unfortunately there is no way to add an `id` prop to it, but we can add an aria label, or remove it completely. 
@ezr-ondrej. WDYT?

Reply:
Lets keep it, cancel is good enough for now.
I'll communicate it to the UX team and decide, we can do follow-up for the `X` button :)